en/python_installation/instructions.md

```diff
@@ -40,7 +40,7 @@ data-collapse=true ces-->
 You need to go to the website https://www.python.org/downloads/release/python-361/ and download the Python installer:
```

Comment:
This URL is for Python 3.6.1 and should probably be changed to conform with `book.py_version`.

Reply:
This one can be changed as follows:

```

You need to go to the website https://www.python.org/downloads/mac-osx/ and download the latest Python installer:

* Download the *Mac OS X 64-bit/32-bit installer* file,
* Double click *python-{{ book.py_release }}-macosx10.9.pkg* to run the installer.

```

So it becomes independent of the version. Users can download whichever version is latest and it won't affect our code.

Reply:
:+1: Good idea.

Reply:
Resolved with commit https://github.com/DjangoGirls/tutorial/pull/1683/commits/e448e25ca043033b7ee91b14482b1aab2b245879